sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -927,4 +927,26 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
       testDefaultColumn
     }
   }
+
+  test("SPARK-50469: V1Writes should respect the output ordering") {
```

Comment:
can we put the test in `SQLInsertTestSuite`? So that we automatically run it three times with file source, v2 tables and hive tables.